src/ORM/Hierarchy/MarkedSet.php

```diff
@@ -180,7 +181,10 @@ public function setNodeCountThreshold($total)
      */
     public function getChildrenMethod()
     {
-        return $this->childrenMethod ?: 'AllChildrenIncludingDeleted';
+        if ($this->childrenMethod) {
+            return $this->childrenMethod;
+        }
+        return $this->rootNode::config()->get('default_children_method');
```

Comment:
Should this be calling `getHierarchyBaseClass()` and getting the config from that?

Reply:
For that matter - is `MarkedSet` restricted to working on `Hierarchy` classes? (I _think_ the answer is a safe yes given we use other `Hierarchy` config in this class already but wanna make sure we're on the same page)

Reply:
Yup restricted to Hierarchy classes, theres an assertion for that already in the constructor

Have added call to getHierarchyBaseClass()